python/cugraph/cugraph/dask/common/input_utils.py

```diff
@@ -19,8 +19,18 @@
 from dask_cudf.core import DataFrame as dcDataFrame
 from dask_cudf.core import Series as daskSeries
 
-import cugraph.comms.comms as Comms
-from raft.dask.common.utils import get_client
+# import cugraph.comms.comms as Comms
+import cugraph.dask.comms.comms as Comms
+# FIXME: this raft import breaks the library if ucx-py is
+# not available. They are necessary only when doing MG work.
+from cugraph.dask.common.read_utils import MissingUCXPy
+try:
+    from raft.dask.common.utils import get_client
+except ModuleNotFoundError as err:
+    if err.name == "ucp":
+        get_client = MissingUCXPy()
+    else:
+        raise ModuleNotFoundError
```

Comment:
This should just be a `raise`, which re-raises the original exception.
```suggestion
        raise 
```
If you do a `raise ModuleNotFoundError`, you lose the original information (mainly the missing module name)